src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/ValidationService.kt

```diff
@@ -648,7 +665,12 @@ class ValidationService(
   private fun validateSupportedSentences(sentencesAndOffences: List<SentenceAndOffences>): List<ValidationMessage> {
     val supportedCategories = listOf("2003", "2020")
     val validationMessages = sentencesAndOffences.filter {
-      !(SentenceCalculationType.isSupported(it.sentenceCalculationType) && supportedCategories.contains(it.sentenceCategory))
+      if (SentenceCalculationType.isSupported(it.sentenceCalculationType) && supportedCategories.contains(it.sentenceCategory)) {
+        val type = from(it.sentenceCalculationType)
+        !featureToggles.botus && type.sentenceClazz == BotusSentence::class.java
+      } else {
+        true
+      }
```

Comment:
Rather than doing this, can you not have `isSupported` set from the Feature toggle in `SentenceCalculationType.kt`?